src/WCSimConstructPMT.cc

```diff
@@ -451,7 +413,7 @@ G4LogicalVolume* WCSimDetectorConstruction::ConstructPMT(G4String PMTName, G4Str
     logicWCPMTsupport2->SetVisAttributes(WCPMTVisAtt_sup);
 
     new G4PVPlacement(0,
-		      G4ThreeVector(0, 0, position_z_offset),
+		      G4ThreeVector(0, 0, 272.36), // value of the z=0 plane with respect to the origin
```

Comment:
CLHEP units? Variable name instead?

Reply:
So this is the placement of the support. I hardcoded it based on the CAD that I have but I am sure we can assign some variable which would have this z value. And since default CLHEP is mm, I didn't show it explicitly.